### DISCUSSION TITLE
LoRA training has only currently been validated for LLaMA models.

### DISCUSSION THREAD

**Discussion by cibernicola** [oobabooga/text-generation-webui#1199] [Q&A]
Hi all!

1st: Great Work

Then question:

Trying to train a LoRA, I'm seeing this message:

"LoRA training has only currently been validated for LLaMA models. Unexpected errors may follow. (Will continue anyway in 5 seconds, press Interrupt to stop.)"

or

"LoRA training does not yet support 4bit. Please use --load-in-8bit for now."

And I'm unable to test it. So the question is pretty obvious, wich exact model is able to be used in order to train a LoRA?

Thanks!

**Comment by BarfingLemurs** [comment]
llama model.
https://huggingface.co/decapoda-research/llama-7b-hf/tree/main

**Reply by EllNiko** [reply]
> llama model. https://huggingface.co/decapoda-research/llama-7b-hf/tree/main

Hi! I downloaded this model, but it is not selected for me if 8 bit is set in the settings. And if I change 8 bit to "none" and load this model, then the LoRA model training does not start. What should I do? I don't understand how to start model training

**Reply by cibernicola** [reply]
> llama model. https://huggingface.co/decapoda-research/llama-7b-hf/tree/main

Downloaded and beging the test, I have 2 files:

![image](https://user-images.githubusercontent.com/4579387/232228760-de4ad2c8-1cd6-4558-8ed2-3b0caef78d19.png)

I'm "merging" the lora in current model, but nothing special happends (in conversation):
![image](https://user-images.githubusercontent.com/4579387/232228849-4bae16da-4518-4d21-ad2a-62b7e56dccb7.png)

Still testing :D thanks for the answer.
